### ISSUE TITLE
TypeScript not recognizing brain.utilities

### ISSUE THREAD

**Issue by DenisTimofijuk** [BrainJS/brain.js#697]
## _What_ is wrong?

Property 'utilities' does not exist on type 'typeof import("f:/Projects/brain.js-snowpack/node_modules/brain.js/src/index")'.ts(2339)

## _How_ do we replicate the issue?

Use TypeScript and run this code:
```
import brain from 'brain.js';
const network = new brain.NeuralNetwork();
const mapScreen = document.getElementById('display-map')!;
mapScreen.innerHTML = brain.utilities.toSVG(network);

```
## Expected behavior (i.e. solution)

Be able not to use //@ts-ignore

## Other Comments

[More details available here](https://github.com/DenisTimofijuk/snowpack-brain.js/blob/main/src/modules/brain-handler.ts)

**Comment by mubaidr** [comment]
Typescript conversion is still in progress, we are looking for contributions: https://github.com/BrainJS/brain.js/issues/684

**Comment by robertleeplummerjr** [comment]
This is no longer an issue, and was resolved around 2.0.0-beta.10 or so.  Just tested with 2.0.0-beta.12 and works perfectly
<img width="579" alt="Screen Shot 2022-04-13 at 10 15 34 AM" src="https://user-images.githubusercontent.com/679099/163200744-06835fe9-2a49-40e5-a01f-931cf31a93d8.png">
.

Sorry for the delay!
